src/browser/declarativewebutils.h

```diff
@@ -42,6 +42,7 @@ public slots:
     QString homePage() const;
     void clearStartupCacheIfNeeded();
     void handleDumpMemoryInfoRequest(QString fileName);
+    void openUrl(QString url);
```

Comment:
Could this be const QString &url and then use tmpUrl inside openUrl by taking it one step higher?

Reply:
Not big and did want to amend while cnanging this. Qt has copy-on-write copy constructor.

Reply:
Personally prefer always const refs but as all were non const didn't start changing them.